packages/block-directory/src/store/reducer.js

```diff
@@ -56,4 +72,5 @@ export const downloadableBlocks = ( state = {
 
 export default combineReducers( {
 	downloadableBlocks,
+	blockManagement,
```

Comment:
Some of the selectors would need to be updated as a result of this change. Some still reference `downloadableBlocks`.